website/app/styles/doc-components/algolia-search/index.scss

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+// ALGOLIA AUTOCOMPLETE SEARCH
+
+// "autocomplete" target container - not used
+// .doc-algolia-search__autocomplete-container {}
+
+// root container of the "autocomplete" injected DOM elements (only the button, in "detached" mode)
+// .aa-Autocomplete {}
```

Comment:
notice: to make it easier to debug and to reference the original styles, I've kept the original naming convention: https://github.com/algolia/autocomplete/blob/next/packages/autocomplete-theme-classic/src/theme.scss

(it's impossible to completely use a custom naming convention, so better to avoid mix & match of two naming conventions)

Reply:
Would we perhaps want to scope the class names as a precaution still? (IE: Add a unique "hds-blah" class name to the outer container then nest other class name references inside so they don't leak.)

Just in the off chance this was ever reused elsewhere such as in an HDS component or even just to establish a convention which we stick to for this kind of thing (similarly to how I scoped the prism-dracula theme styles used by doc-code-block so they wouldn't leak and effect the HDS CodeBlock).